frontend/src/Header.js

```diff
@@ -8,12 +8,20 @@ function Header() {
 
     const {user, setUser} = useContext(AuthContext);
 
+    const profileClass = window.location.pathname.match(/.profile/) ? "active" : "";
+    const customersClass = window.location.pathname==="/customers" ? "active" : "";
+
     return (
         <Navbar fixed="top" collapseOnSelect expand="lg" variant="dark" className="header">
             <Navbar.Toggle aria-controls="responsive-navbar-nav"/>
             <Navbar.Collapse id="responsive-navbar-nav">
                 <Nav style={{height: '45px'}} className="mr-auto">
-                    {user && user.currentCustomerId ? <Nav.Link href="/profile">Profile</Nav.Link> : null}
```

Comment:
user && user.currentCustomerId is encountered in several places in the component. Can be extracted in a separate variable and used instead.